README.md

```diff
@@ -43,6 +43,7 @@ Currently supported grammars are:
   * Python
   * RSpec
   * Racket
+  * [RANT](https://github.com/TheBerkin/Rant)
```

Comment:
No worries. It's such a small and specialized language (procedural text generation) that I wanted to give you a reference point to work from, in case you hadn't heard of it.